README.md

```diff
@@ -30,6 +30,25 @@ pip install -r requirements.txt
 npm install
 ```
 
+## Database creation
+
+To instantiate the items table and add an item, run this Python code.
+
+```python
```

Comment:
Let's figure out how to get this into a migration and then tell people to run a migration.